rfd/0084-azure-backend.md

```diff
@@ -0,0 +1,130 @@
+---
+authors: Edoardo Spadolini (edoardo.spadolini@goteleport.com)
+state: draft
+---
+
+# RFD 0084 - Azure backend storage
+
+## Required Approvers
+* Engineering: TBD
+* Security: TBD
+* Product: TBD
+
+## What
+
+Add support for Azure IAM authentication to the Postgres backend, add audit log capabilities to the `sqlbk`/Postgres backend, add a session uploader that uses Azure Blob Storage (with the same IAM authentication).
```

Comment:
The real "what" here is just to use Azure services as a backend. The rest of the info here is more about the "how". Suggested edit:

```suggestion
Enable users to leverage Azure-native services as a Teleport backend.
We will accomplish this by:
- Add support for Azure IAM authentication to the existing Postgres backend
- Add audit log capabilities to the `sqlbk`/Postgres backend
- Add a session uploader that uses Azure Blob Storage (with the same IAM authentication)
```